src/DynamoCore/Library/ILibraryLoader.cs

```diff
@@ -11,5 +12,7 @@ public interface ILibraryLoader
         /// Loads node's library
         /// </summary>
         void LoadNodeLibrary(Assembly library);
+
+        void LoadPackages(IEnumerable<Assembly> assemblies);
```

Comment:
we can't modify an interface like this. API break.

Reply:
🤦‍♂